jetstream/analysis.py

```diff
@@ -642,6 +642,13 @@ def run(self, current_date: datetime, dry_run: bool = False) -> None:
 
     def ensure_enrollments(self, current_date: datetime) -> None:
         """Ensure that enrollment tables for experiment are up-to-date or re-create."""
+        if (
+            hasattr(self.config.experiment, "is_enrollment_paused")
+            and not self.config.experiment.is_enrollment_paused
```

Comment:
If this gets set through a config file jetstream should still run the analysis?

Reply:
Yea good point. Probably not the right place to be checking for this (or maybe not the only place). I'll make some updates to the PR.